packages/cli/src/modules/insights/insights.config.ts

```diff
@@ -43,4 +43,18 @@ export class InsightsConfig {
 	 */
 	@Env('N8N_INSIGHTS_FLUSH_INTERVAL_SECONDS')
 	flushIntervalSeconds: number = 30;
+
+	/**
+	 * How old (days) insights data must be to qualify for regular deletion
+	 * Default: -1 (no pruning)
```

Comment:
Maybe there was some discussion already about this, but I'm uneasy defaulting this to disabled.

Reply:
In the RFC, you mention that the compaction mechanism is made to keep the storage size under control, that's why it's default to -1.
But I might have missed overriding this value with the license one, because I see a quota for maxAgeDays for inside in the cloud licenses

Reply:
I did the benchmarking but the RFC was mostly Danny :)

In any case, if the team is satisfied, then all good.